DuckDuckGo/HomePage/View/HomePageViewController.swift

```diff
@@ -271,6 +275,60 @@ final class HomePageViewController: NSViewController {
             .show(in: view.window)
     }
 
+    private func showSettingsOnboardingIfNeeded() {
+        if !settingsVisibilityModel.didShowSettingsOnboarding {
+            DispatchQueue.main.async {
```

Comment:
❓ Out of curiosity, is this dispatch necessary? Did you see some race conditions when executing this immediately when `viewDidAppear` was called?

Reply:
Yes, the view frame was incorrect 😬 I'll add a comment about it.